### PR TITLE
Switch to URLs for navigation between calls

### LINKED ISSUE #13: Browser back and forward buttons don't work
The html frontend should build a history of visited function calls during a debugging session.

### PR DESCRIPTION
**Pull request by pepeiborra**
Fixes #13

### CODE REVIEW

File: elm/Main.elm [hunk 1]
```diff
@@ -258,7 +262,7 @@ boundedPager radius f p =
 
 viewCallLink : Int -> String -> Html Msg
 viewCallLink index rendered =
-    a [ href "javascript:()", onClick (SelectCall index) ] [ text rendered ]
+    a [ href ("#call/" ++ toString index) ] [ text rendered ]
```
Comment (ndmitchell) [inline elm/Main.elm]: Do we need any fancy escaping here? Does it still work if you have `/` as the name or similar?

Comment (pepeiborra) [reply elm/Main.elm]: Not sure, but I would assume that Elm takes care of escaping the `rendered` text child. Or are you thinking of the href link ?

Comment (ndmitchell) [reply elm/Main.elm]: Yes, the href link, which might need different escaping with URL encoding for certain characters (or might not - happy to wait til we see the problem).

Comment (pepeiborra) [reply elm/Main.elm]: Well, `index` is an int, so provided the type system doesn't lie, I think no injection is possible here right?

Comment (ndmitchell) [reply elm/Main.elm]: Oh, I hadn't noticed that - makes sense - an Int guarantees it is safe.

**Pull request merged**